localstack/ext/java/src/test/java/cloud/localstack/S3UploadTest.java

```diff
@@ -24,19 +25,39 @@
 @RunWith(LocalstackTestRunner.class)
 public class S3UploadTest {
 
+	/**
+	 * Test based on https://github.com/localstack/localstack/issues/359
+	 */
 	@Test
-	public void testUpload() throws Exception {
-		/*
-		 * Test based on https://github.com/localstack/localstack/issues/359
-		 */
+	public void testTrival() throws Exception  {
+		testUpload("{}"); // Some JSON content, just an example
+	}
+
+	/**
+	 * Test will fail because it is greater than 128k
```

Comment:
Please give it a try with the latest version, then update or remove these comments @jhovell .